spec/models/ingredient_spec.rb

```diff
@@ -12,4 +12,17 @@
       it { should have_many(:recipes).through(:recipe_ingredients) }
    end
 
+
+   describe "class methods" do
+      describe "#calulate_cost" do
+         it "calculates cost of all ingredients" do
+            beef = Ingredient.create!(name: "Ground Beef", cost: 2)
+            salt = Ingredient.create!(name: "Salt", cost: 4)
+            penuts = Ingredient.create!(name: "Penut", cost: 7)
+
+            expect(Ingredient.calculate_cost).to eq(13)
```

Comment:
This test works because you made `calculate_cost` a class method. However you're also using it in the view as part of `@recipe.ingredients`, so it might be good to test that usage in this model test as well. 
Bottom line, it's important that we test model methods in the same way (or ways) that we use them in the actual application.